systers_portal/templates/meetup/meetup.html

```diff
@@ -13,5 +14,14 @@ <h2>{{ meetup.title }}</h2>
     </p>
     <hr>
     {{ meetup.description|safe }} <br/>
+
+    <div class="share-buttons">
+        <p> Share this: </p>
+        <span class='st_facebook_large' displayText='Facebook'></span>
+        <span class='st_twitter_large' displayText='Tweet'></span>
```

Comment:
We need to add the Twitter account `@systers_org` in tweet message. Can you please enable this? 

Also it looks better to say *Share on X*, like *Share on Twitter*.

Everything else looks, 👍 Thanks!